pkg/kotsadm/objects/postgres_objects.go

```diff
@@ -31,7 +31,7 @@ func PostgresStatefulset(deployOptions types.DeployOptions, size resource.Quanti
 		}
 	} else if deployOptions.KotsadmOptions.OverrideRegistry != "" {
 		// if there is a registry specified, use the postgres image there and not the one from docker hub - even though there's not a username/password specified
-		image = fmt.Sprintf("%s/postgres:%s", kotsadmversion.KotsadmRegistry(deployOptions.KotsadmOptions), kotsadmversion.KotsadmTag(deployOptions.KotsadmOptions))
+		image = fmt.Sprintf("%s/postgres:%s", kotsadmversion.KotsadmRegistry(deployOptions.KotsadmOptions), strings.Split(image, ":")[1])
```

Comment:
@laverya tag here should always be the original postgres tag, right?

Reply:
I believe so, but I know it's been the kotsadm tag at certain times in the past.  It needs to match the code in [push-images](https://kots.io/kots-cli/admin-console/push-images/), for sure. (and if you make it use the actual upstream tag, you'll need to ensure that CI pushes that tag to ttl.sh and not `2h`)